Formula/rxvt-unicode.rb

```diff
@@ -39,6 +46,17 @@ class RxvtUnicode < Formula
   end
 
   def install
+    ENV.cxx11
+    resource("libptytty").stage do
+      system "cmake", "-S", ".", "-B", "build", *std_cmake_args, "-DBUILD_SHARED_LIBS=ON"
```

Comment:
Installing this into `prefix` feels like asking for `brew link` conflicts.

Reply:
That's a good catch. I copied this from the previous PR but I agree it should go in `libexec` instead.

Reply:
Why not add this as a separate formula? Or, if not a separate formula: why not just build the static libraries and just install them into `buildpath` so we don't have to carry them around?

They're already going to be put in `libexec` which limits their reusability; may as well avoid the overhead of shared linking while we're doing that.

Reply:
It doesn't meet the notability requirements to be a formula so that might not be the best route.  The static library sounds fine to me.  It's not actually clear that any software other `rxvt-unicode` had much if any use for this.  I think I only have to append the `buildpath` (or, perhaps, a subdirectory of `buildpath`) to the `LDFLAGS` for the static library to be found.

Reply:
I pushed a change to build `libptytty` as a static library in the `buildpath` and link it from there.  I don't really know why this is even a separate library right now, but if for some reason another formula eventually did want to use `libptytty`, we could revisit making it a standalone formula.

Reply:
> It doesn't meet the notability requirements to be a formula

This doesn't apply to things that maintainers want to add to make things cleaner or for our (Homebrew's) own usage.